packages/components/src/tabs/styles.ts

```diff
@@ -24,7 +24,7 @@ export const TabListWrapper = styled.div`
 
 	@media not ( prefers-reduced-motion: reduce ) {
 		&.is-animation-enabled::after {
-			transition-property: left, top, width, height;
+			transition-property: inset-inline-start, top, width, height;
```

Comment:
This change is necessary so that the indicator animations continue to work as before.